src/screens/CreatePost/index.js

```diff
@@ -89,7 +90,7 @@
   const sheetBackRef = React.useRef();
 
   const [typeUser, setTypeUser] = React.useState(false);
-  const {headerTitle, initialTopic, isInCreatePostTopicScreen, anonUserInfo} =
+  const {headerTitle, initialTopic, isInCreatePostTopicScreen, anonUserInfo, setSelectedTopic} =
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setSelectedTopic` state setter is now being returned from `useCreatePostHook`. Ensure that the hook is properly managing and updating this state, as improper handling could lead to unexpected behavior in your components.
```diff
+ const {headerTitle, initialTopic, isInCreatePostTopicScreen, anonUserInfo, setSelectedTopic} = useCreatePostHook(typeUser);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->